packages/split-button/package.json

```diff
@@ -0,0 +1,43 @@
+
+  {
+    "name": "@leafygreen-ui/split-button",
+    "version": "0.9.0",
+    "description": "leafyGreen UI Kit Split Button",
+    "main": "./dist/index.js",
+    "module": "./dist/esm/index.js",
+    "types": "./dist/index.d.ts",
+    "typesVersions": {
+      "<3.9": { "*": ["ts3.4/*"] }
+    },
+    "scripts": {
+      "build": "../../node_modules/.bin/rollup --config ../../rollup.config.js"
+    },
+    "license": "Apache-2.0",
+    "homepage": "https://github.com/mongodb/leafygreen-ui/tree/main/packages/split-button",
+    "repository": {
+      "type": "git",
+      "url": "https://github.com/mongodb/leafygreen-ui"
+    },
+    "bugs": {
```

Comment:
nit: is this still something we use?

Reply:
it looks like it correctly links to the latest LG tickets in jira

Reply:
Should we add a ticket to update this in the template?

Reply:
what would we update it to? i can update it in this PR